spec/models/group_spec.rb

```diff
@@ -0,0 +1,51 @@
+RSpec.describe Group do
+  required_props = %i(
+    name
+    moderator_email
```

Comment:
@Enriikke do you think maybe this should be handled with an association instead? The idea is that the User that creates the group becomes the moderator. Perhaps I want to add a `belongs_to` association to the Group instead of this attribute?